### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/advanced/seo.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#496] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/advanced/seo.html

vuepress-theme-hope 通过内置 为提供 SEO 增强功能。 为了使插件能够更好的工作，你可能需要查看一下 页面配置 并合理的配置它们。 相关信息 vuepress-theme-hope 将主题选项中的 plugins.seo 作为插件选项提供给 vuepress-plugin-seo2。 插件会通过向网站 <head> 注入标签，让你的网站完全支持 开放内容协议 OGP 和 JSON-LD 1.1，以全面增强站点的搜索引擎优化性。 如果不需要这个插件，请设置在主题选项中设置 plugins.seo: false。

https://theme-hope.vuejs.press/zh/guide/advanced/seo.html

**Comment by GodX-18** [comment]
1

**Comment by GodX-18** [comment]
d

**Comment by javaweb2019** [comment]
这个好？

**Comment by 006hjy** [comment]
请问如何在生成的每个html的head标签前插入百度统计的js代码？

**Reply by Mister-Hope** [reply]
有一个官方插件叫。@vuepress/plugin-baidu-analyze
